spec/src/utils/request-queue.js

```diff
@@ -356,13 +356,15 @@ describe('ConstructorIO - Utils - Request Queue', () => {
         requests2.send();
 
         setTimeout(() => {
+          expect(sendSpy1.callCount).to.be.at.least(2 + 1); // 2 min sent + 1 finally
+          expect(sendSpy2.callCount).to.be.at.least(2 + 1); // 2 min sent + 1 finally
           expect(sendSpy1.callCount + sendSpy2.callCount).to.equal(5 + 2); // 5 sent + 2 finally
           expect(RequestQueue.get()).to.be.an('array').length(0);
           done();
         }, waitInterval);
       });
 
-      it('Should send tracking requests when items are queued using only one instance and user is human', (done) => {
+      it('Should send tracking requests from both queues when items are queued in one and user is human', (done) => {
```

Comment:
similar to above maybe reword to 'Should send tracking requests when using both queues where items are queued in one...'